seo.php

```diff
@@ -70,10 +81,10 @@ function fetchFromSeoApi($key, $url){
     $SEO_json = fetchFromSeoApi($key, $url);
 
     // Set SEO <head> information for index.php
-    $SEO_head = $SEO_json->html->head;
+    $SEO_head = isset($SEO_json->html->head) ? $SEO_json->html->head : "";
```

Comment:
what about if $SEO_json->html is already empty? does it break? (with default seoapi response)